ccd-definition/CaseTypeTab/Documents.json

```diff
@@ -33,5 +33,14 @@
     "TabDisplayOrder": 4,
     "CaseFieldID": "servedDocumentFiles",
     "TabFieldDisplayOrder": 4
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "TabID": "Documents",
+    "TabLabel": "Claim documents",
+    "TabDisplayOrder": 4,
+    "CaseFieldID": "manageDocuments",
```

Comment:
I believe this is what causing the issue in Highleveldatadetup step.
Looks like this needs to go into a CUI-nonprod.json file as well. Otherwise only this gets uploaded and nothing else in ccd (As CUI non prod files are all ignored) So data setup fails.

Reply:
moved the changes to new file as mentioned